### DISCUSSION TITLE
where is the config 'max.poll.records'

### DISCUSSION THREAD

**Discussion by La1a1a** [confluentinc/confluent-kafka-python#1385] [Q&A]
I use confluent-kafka to replace kafka-python, build a consumer[use DeserializingConsumer(it doesn't matter) ]. and user consumer.poll() to do sth
How I set the config'max.poll.records'.... when I use the old kafka, I can set 'max_poll_records'...but now, i can't find it in any docs

**Comment by OneCricketeer** [comment]
It's in `consume` with `num_messages`, but it is [not implemented](https://github.com/confluentinc/confluent-kafka-python/blob/master/src/confluent_kafka/deserializing_consumer.py#L126-L132)

The poll function only returns one record at a time
